package.json

```diff
@@ -58,7 +58,14 @@
     "webpack-dev-server": "^3.5.1"
   },
   "dependencies": {
+    "axios": "^0.19.0",
+    "dotenv": "^8.0.0",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-redux": "^7.0.3",
+    "react-router-dom": "^5.0.1",
+    "redux": "^4.0.1",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
I thought this is dev dependency. 
Please add these dev dependencies to devDependancies. 
e.g `npm install --save-dev redux-devtools-extension`.